framework/auth/exceptions.py

```diff
@@ -60,3 +60,19 @@ class ExpiredTokenError(EmailConfirmTokenError):
     """Raised if an email confirmation token is expired."""
     message_short = "Expired Token"
     message_long = language.EXPIRED_EMAIL_CONFIRM_TOKEN
+
+
+class RetractionTokenError(FrameworkError):
```

Comment:
Retraction has to do with nodes, so this should go where the other node-related exceptions are.